### PR TITLE
5103 Fixed the checkbox to check on label

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#5103

*Description of changes:*

- Fixed the checkbox to check on label

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Articles/CompanyProvisions.vue [hunk 1]
```diff
@@ -229,15 +223,22 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
   margin-left: 0;
 }
 
-::v-deep .v-input--selection-controls {
-  margin-top: 1.875rem;
-  padding: 0;
-}
-
-.checkbox-label {
+::v-deep .v-input--checkbox .theme--light.v-label {
```
Comment (severinbeauvais) [inline src/components/Articles/CompanyProvisions.vue:226]: Do you still need the styling on line 220 above?

File: src/components/Articles/CompanyProvisions.vue [hunk 2]
```diff
@@ -229,15 +223,22 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
...
   font-weight: normal;
+}
+
+::v-deep .invalid.v-input--checkbox .theme--light.v-label {
+  color: $app-red;
+}
+
+::v-deep #checkbox-div .v-input__slot {
+  align-items: start;
+  justify-content: start;
```
Comment (severinbeauvais) [inline src/components/Articles/CompanyProvisions.vue:238]: Are you sure you need all this styling for the checkbox/label? I don't recall having to do this with other checkboxes. Please have another look (or explain the problem). Thx.

Comment (andrepestana-aot) [reply src/components/Articles/CompanyProvisions.vue:238]: The problem is keeping the checkbox to the top as UX required. 
image

Comment (severinbeauvais) [reply src/components/Articles/CompanyProvisions.vue:238]: OK, good explanation :)

File: src/components/Articles/CompanyProvisions.vue (hunk 1 above)
Comment (andrepestana-aot) [reply src/components/Articles/CompanyProvisions.vue:226]: Yes, I do need them.

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**